react-app/src/Home.js

```diff
@@ -1,21 +1,154 @@
 import { StatusBar } from 'expo-status-bar'
-import { StyleSheet, View, Button } from 'react-native'
+import { StyleSheet, View, FlatList, Text, Pressable } from 'react-native'
 import NavBar from './NavBar.js'
 import React from 'react'
+import { DUMMY_DATA } from '../data/DummyData'
+import { SvgXml } from 'react-native-svg'
+import ClockIcon from '../assets/icons/clock.svg'
+import ArrowIcon from '../assets/icons/arrow-right.svg'
 
-export default function Home({ navigation }) {
+// Component of the card
+function Card({ title, description, image, available, type }) {
+  return (
+    <View style={styles.cardContainer}>
+      <ImageInCard image={image} type={type} style={styles.imageContainer} />
+      <View style={styles.cardLowerPart}>
+        <TextInCard title={title} description={description} available={available} />
+        <Press></Press>
+      </View>
+    </View>
+  )
+}
+
+// Component of the image in the card
+function ImageInCard({ image, type }) {
+  return (
+    <View
+      style={[
+        styles.imageContainer,
+        type == 'standalone' ? styles.standaloneCardColor : styles.journeyCardColor
+      ]}
+    >
+      <View style={styles.svgContainer}>
+        <SvgXml width='100%' height='100%' xml={image} />
+      </View>
+    </View>
+  )
+}
+
+// Component of the text in the card
+function TextInCard({ title, description, available }) {
+  return (
+    <View style={styles.textContainer}>
+      {!available && (
+        <Text style={styles.status}>
+          <ClockIcon width={16} height={16} /> coming soon
+        </Text>
+      )}
+      <Text style={[styles.textTitle, !available && styles.unavailable]}>{title}</Text>
+      {description?.map((value, index) => (
+        <Text key={index} style={!available && styles.unavailable}>
+          {' '}
+          - {value}
+        </Text>
+      ))}
+    </View>
+  )
+}
+
+// Component of the button in the card
+function Press() {
```

Comment:
Rename it to ButtonInCard may be a better idea to align other namings?

Reply:
Good idea.